### DISCUSSION TITLE
Q: Temporarily disable integration?

### DISCUSSION THREAD

**Discussion by MehrCurry** [Hyundai-Kia-Connect/kia_uvo#965] [Q&A]
Since a few days I ran into trouble with the Hyundai rate limits. The rate limit strikes early in the morning. It was working for years. Also have EVCC running. I want to disable both temporarily to see what is most likely causing those problems? So is it possible to disable this integration without removing it completely?

**Comment by cdnninja** [comment]
Yes. All integrations can be disabled from the integration screen.

**Comment by MehrCurry** [comment]
Silly question. Found the three dots.

Thanks
